package.json

```diff
@@ -37,12 +37,13 @@
   },
   "dependencies": {
     "@types/bluebird": "3.5.18",
-    "@types/dockerode": "^2.5.1",
+    "@types/dockerode": "2.5.5",
```

Comment:
Aligning this with `resin-builder`:
https://github.com/balena-io/resin-builder/blob/master/package.json#L20
The absence of the caret (`^`) is intentional, see Page's pull request:
https://github.com/balena-io/resin-builder/commit/d09e63603b967b57470e93e9e7f76f4d6eaeaf20